### ISSUE TITLE
App Crashes

### ISSUE THREAD

**Issue by OlegTarashkevich** [romandanylyk/PageIndicatorView#18]
The app crashes after you last update. Please increase maven version, don't rewrite previous version with the same number. Now I should temporary disabled you lib from my projects ((.
I use different view ids.

Please look at the attached screenshots:
![screen shot 2017-02-03 at 13 04 29](https://cloud.githubusercontent.com/assets/13828110/22587126/6315107a-ea11-11e6-8e14-6930b4bdc6f7.png)
![screen shot 2017-02-03 at 13 04 38](https://cloud.githubusercontent.com/assets/13828110/22587125/631351c2-ea11-11e6-8ca7-e1da4918144c.png)

@romandanylyk ^^

**Comment by romandanylyk** [comment]
Hi
what do you mean by increasing maven version?
After releasing a new version I do make a tag about it, so can track it in git. 
https://github.com/romandanylyk/PageIndicatorView/commit/0ec800970aff9bbfc9273ab3cf7312fd17897f91

**Comment by OlegTarashkevich** [comment]
@romandanylyk Thank you for your feedback. 
I see that you increased maven version, my bad.
I used 0.0.9 and all works properly, then I've updated to 0.1.0 and see crashes. It's appears on activity that have rotation. Now I have 85 crashes. 
You can launch this app https://play.google.com/store/apps/details?id=com.lucky.notewidget and open first screen (help screen).
I will try to use previous version.

```
Fatal Exception: java.lang.RuntimeException
Unable to start activity ComponentInfo{com.lucky.notewidget/com.lucky.notewidget.ui.activity.HelpActivity}: java.lang.IllegalArgumentException: Wrong state class, expecting View State but received class com.rd.SavedState instead. This usually happens when two views of different type have the same id in the same hierarchy. This view's id is id/help_indicator. Make sure other views do not use the same id.
```

**Comment by azmifadz** [comment]
fixed after upgrading to 0.1.1 . Thanks a lot!
